app/budget/directives/budgetNav/budgetNav.css

```diff
@@ -1,16 +1,38 @@
 .budget-nav {
 	display: flex;
 	justify-content: space-between;
+	background-color: #F8F8F8;
+	border: 1px solid #CCCCCC;
+	margin-bottom: 25px;
+	border-radius: 3px;
 }
 
 .nav-items {
 	display: flex;
 }
 
```

Comment:
All styles in this file should be prefixed with .budget-nav to avoid them potentially affecting different components that use the same class names/selectors

Example:
.nav-items {
Should instead be:
.budget-nav .nav-items {